File: RECORRIDO SECUENCIAL/recorridosecuencial.cs
using System;
class Program {
    static void Main(string[] args) {
        int[] arr = {30, 40, 50, 60, 70, 80};
        Console.WriteLine("recorrido secuencial:");
        for (int e = 0; e < arr.Length; e++) {
            Console.WriteLine(arr[e]);
        }
    }
}

File: RECORRIDO SECUENCIAL/recorridosecuencial.cpp
#include <iostream>
using namespace std;
int main () {
    int arr[6] = {30, 40, 50, 60, 70, 80};
    cout << "recorrido secuencial:" << endl;
    for (int a = 0; a < 6; a++) {
        cout << arr[a] << endl;
    }
    return 0;
}